TruthTables/Tasks.qs

```diff
@@ -44,19 +44,23 @@ namespace Quantum.Kata.TruthTables {
 
     // Task 1. Projective functions (elementary variables)
     //
-    // Goal: Describe the three projective functions x₁, x₂, x₃ as 3-input
-    //       functions, represented by integers.  Note that we follow the
-    //       convention that x₁ is the least-significant input.
+    // Goal: Describe the three projective functions f₁, f₂, f₃ represented
+    // by integers. They are 3 input functions i.e. f(x) : {0,1}ⁿ → {0,1},
```

Comment:
You can write this more concisely using Unicode (my go-to page for task descriptions is https://en.wikipedia.org/wiki/Unicode_subscripts_and_superscripts):
```suggestion
    // by integers. They are 3 input functions i.e. f(x) : {0,1}³ → {0,1}.
```